frontends/concrete-python/examples/levenshtein_distance/levenshtein_distance.py

```diff
@@ -41,6 +41,7 @@ def dna():
 
     def __init__(self, letters: str):
         self.letters = letters
+        self.mapping_to_int = {}
```

Comment:
BTW you can remove them from the top level class. Or leave just the type

Reply:
Indeed, done